property/list.go

```
@@ -11,18 +11,15 @@ import (
 )
 
 type ListResult common.BaseListResultType[common.Property]
-type FilterParams struct {
-	Slug string
-}
-
-var propertyListResult ListResult
 
 // List lists the properties from a given Organization.
 // Edgio's list page size was defaulted to 100 for now,
 // which is the highest value. The idea is to return all properties
 // until actual pagination is implemented.
 // Returns a list of properties for a given Organization, or an error if anything goes wrong.
 func (c ClientStruct) List() (ListResult, error) {
+	var propertyListResult ListResult
```

Hey @rafapaulin I was checking your code on the whole function for context and realized I missed one thing multiple times on my reviews that might be improved for the future for all the errors you are handling, like in lines 28,32,37,42, 47 and 60 (probably all over the place too, did not checked it)

e.g for context:
```go
parsedURL, err := url.Parse(serviceURL)
if err != nil {
        return ListResult{}, errors.New(err.Error())
}
```

the thing is that you can simply return the error as it is since `err.Error()` returns a string, and `errors.New()` creates a new error from a string. If err is already an error, there is no need to convert it to a string and then back to an error, this ends up being redundant.